menu/cakemenu.html

```diff
@@ -0,0 +1,270 @@
+										<article>
+											<a href="#" class="image"><img src="images/cakes/cake_pic02.jpg" alt="thanksgiving pie" /></a>
+											<h3>Holidays</h3>
+											<p>Aenean ornare velit lacus, ac varius enim lorem ullamcorper dolore. Proin aliquam facilisis ante interdum. Sed nulla amet lorem feugiat tempus aliquam.</p>
+											<ul class="actions">
+												<li><a href="orders.html" class="button prefill-button" data-order-type="holiday">Order This</a></li>
+											</ul>
+										</article>
+										<article>
+											<a href="#" class="image"><img src="images/cakes/cake_pic14.jpg" alt="company anniversary cake" /></a>
+											<h3>Corporate Events</h3>
+											<p>Aenean ornare velit lacus, ac varius enim lorem ullamcorper dolore. Proin aliquam facilisis ante interdum. Sed nulla amet lorem feugiat tempus aliquam.</p>
+											<ul class="actions">
+												<li><a href="orders.html" class="button prefill-button" data-order-type="corporate-event">Order This</a></li>
+											</ul>
+										</article>
+										<article>
+											<a href="#" class="image"><img src="images/cakes/cake_pic13.jpg" alt="birthday cake" /></a>
+											<h3>Birthdays</h3>
+											<p>Aenean ornare velit lacus, ac varius enim lorem ullamcorper dolore. Proin aliquam facilisis ante interdum. Sed nulla amet lorem feugiat tempus aliquam.</p>
+											<ul class="actions">
+												<li><a href="orders.html" class="button prefill-button" data-order-type="birthday">Order This</a></li>
+											</ul>
+										</article>
+									</div>
+									<footer id="footer" class="align-center cta">
+										<h3>Have an event or idea that is not listed here?</h3> 
+										<p>Fill out the <a href="orders.html">order form</a> or <a href="mailto:amandabe723+personalsite@gmail.com">email me</a> to discuss whether you can have your cake and eat it.</p>
+									</footer>
+								</section>
+
+						</div>
+					</div>
+
+				<!-- Sidebar -->
+					<div id="sidebar">
+						<div class="inner">
+
+							<!-- Search -->
+								<!-- <section id="search" class="alt">
+									<form method="post" action="#">
+										<input type="text" name="query" id="query" placeholder="Search" />
+									</form>
+								</section> -->
+
+							<!-- Menu -->
+								<nav id="menu">
+									<header class="major">
+										<h2>Menu</h2>
+									</header>
+									<ul>
+										<li><a href="../index.html">Homepage</a></li>
+										<li><a href="../about.html">About</a></li>
+										<li>
+											<span class="opener">Menu</span>
+											<ul>
+												<li><a href="cakemenu.html" id="navbar-selected">Cake</a></li>
+												<li><a href="cupcakemenu.html">Cupcakes</a></li>
+												<li><a href="piesmenu.html">Pies</a></li>
+												<li><a href="cookiesmenu.html">Cookies</a></li>
+											</ul>
+										</li>
+										<li><a href="../gallery.html">Gallery</a></li>
+										<li>
```

Comment:
but since gallery.html isn't in the same level as the cakemenu.html (or other _menu.html's for that matter), we have to add "../" to get out of one folder

Reply:
thanks for explaining/reminding about this